lib/pci.py

```diff
@@ -0,0 +1,480 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2019 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+
+"""
+Module for all PCI devices related functions.
+"""
+
+
+import re
+import os
+import commands
```

Comment:
Have a wrapper function to execute commands and return output/fail out in error cases and use it across,
will be helpful when we have to adhere to python3 etc...